#!/bin/sh -e
## 03_xpm_icon.dpatch by Rene Engelhard <rene@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use xpm and 32x32 to conform with the menu policy

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/rene/fortified-0.9.2/pixmaps/fortified.xpm fortified-0.9.2/pixmaps/fortified.xpm
--- /home/rene/fortified-0.9.2/pixmaps/fortified.xpm	1970-01-01 01:00:00.000000000 +0100
+++ fortified-0.9.2/pixmaps/fortified.xpm	2003-06-12 23:53:50.000000000 +0200
@@ -0,0 +1,295 @@
+/* XPM */
+static char *fortified[] = {
+/* columns rows colors chars-per-pixel */
+"32 32 257 2",
+"   c #005B005C003D",
+".  c #071D098504B7",
+"X  c #07FD08550843",
+"o  c #0CB605750537",
+"O  c #0957079C0811",
+"+  c #0AEA0BFE061B",
+"@  c #0B8E0C5709EA",
+"#  c #0BD1131.1.02",
+"$  c #0EAA11C60A3B",
+"%  c #154B0A4F0962",
+"&  c #137014030BDE",
+"*  c #166519510BFD",
+"=  c #1A561CD80F10",
+"-  c #11321AEB0762",
+";  c #12C1127411CD",
+":  c #1B751D111464",
+">  c #1BE01ABE1802",
+",  c #15CC25D2091F",
+"<  c #1BAA229D0CBC",
+"1  c #1BD52BCE0CB3",
+"2  c #1D23227A121E",
+"3  c #1DCF32760BEF",
+"4  c #28690C2E0BC3",
+"5  c #349E0F340F12",
+"6  c #229F2608138C",
+"7  c #24E72B7F155B",
+"8  c #282B27521B7C",
+"9  c #25E03CEA0F7A",
+"0  c #224732DF0F61",
+"q  c #2C0634291A1D",
+"w  c #2A3E3DF311F2",
+"e  c #2E8B3BE61D51",
+"r  c #260C34831051",
+"t  c #37842AE81630",
+"y  c #331438B01AC2",
+"u  c #2C192BDB205A",
+"i  c #2DDC332022ED",
+"p  c #311D2D0125F4",
+"a  c #327E36A123C5",
+"s  c #36EC3D9B253E",
+"d  c #394D3A7B2A8A",
+"f  c #391638133740",
+"g  c #3B063D073193",
+"h  c #264942520F97",
+"j  c #2F5548A013B8",
+"k  c #397C4CCB1D12",
+"l  c #348F478D184F",
+"z  c #382154D1179C",
+"x  c #3ADC529A1C0A",
+"c  c #3CA24BAC2388",
+"v  c #3A8B43E126EA",
+"b  c #3E1451.1.090",
+"n  c #3F6760C01C28",
+"m  c #42570FDD0E5C",
+"M  c #4C8210F20FD4",
+"N  c #4D5B12C61309",
+"B  c #5C6612EE11CA",
+"V  c #52632C481DED",
+"C  c #5E6B3DAE1D51",
+"Z  c #45DE2F582F5C",
+"A  c #47C83F862076",
+"S  c #525F25822232",
+"D  c #48853D293804",
+"F  c #6F5B19480EEE",
+"G  c #6F441D031087",
+"H  c #6E9919241888",
+"J  c #76D710B00F80",
+"K  c #7D5B1E381BA5",
+"L  c #747B1A171682",
+"P  c #6CF325D11141",
+"I  c #6EF039851CD6",
+"U  c #771420621C63",
+"Y  c #770A39371E2A",
+"T  c #772639852123",
+"R  c #43CD42A918FD",
+"E  c #40F355D21AA8",
+"W  c #42774EBA231A",
+"Q  c #4950491631FC",
+"!  c #476153C8268E",
+"~  c #46C353F12D74",
+"^  c #4D3850C925A1",
+"/  c #49E35C2D2817",
+"(  c #4D8556A43245",
+")  c #5E584C6926A0",
+"_  c #52AD42993DCB",
+"`  c #57905C352C3C",
+"'  c #52785D993668",
+"]  c #51BD5FDE3B96",
+"[  c #5C4D57EA3567",
+"{  c #4A5B65671AF0",
+"}  c #484368DE1D6C",
+"|  c #545474FA1CCB",
+" . c #4C39676B2442",
+".. c #4AF6677525B5",
+"X. c #4D627D8320BE",
+"o. c #5C236E632C54",
+"O. c #56FA6FB72CF2",
+"+. c #592D6A0D3B43",
+"@. c #527074AB2346",
+"#. c #5436708D29F5",
+"$. c #54147B8E2453",
+"%. c #59D0767E2950",
+"&. c #5B707D81294D",
+"*. c #55F97C7E2EAB",
+"=. c #5ED37CF33409",
+"-. c #705F5A451E57",
+";. c #703942BC22B1",
+":. c #64CA5DBE2D90",
+">. c #633658053A9F",
+",. c #7DA95B6037B9",
+"<. c #78AB487824F9",
+"1. c #6FF363B3201B",
+"2. c #62FA67372B9D",
+"3. c #654D65B53904",
+"4. c #656C78352AB3",
+"5. c #65237BB42765",
+"6. c #60F3785C3146",
+"7. c #615D7CE93C31",
+"8. c #69707866313B",
+"9. c #6C70773B3D42",
+"0. c #6A387A9F3105",
+"q. c #7F957CA53AAF",
+"w. c #465745834186",
+"e. c #55844BBC4503",
+"r. c #5E3754D143FA",
+"t. c #56925A1B4572",
+"y. c #51EB57D85092",
+"u. c #5B735B0C590C",
+"i. c #5EA366AD4778",
+"p. c #5D936C844694",
+"a. c #581B722E49FF",
+"s. c #63C26C264499",
+"d. c #640875C14EFC",
+"f. c #6A7276B947C4",
+"g. c #6EE471D454F3",
+"h. c #7F656E714811",
+"j. c #7843692A49BF",
+"k. c #7DEE7DEE4D61",
+"l. c #7F287B2E5B5C",
+"z. c #64FE67D05477",
+"x. c #70BB6EF16D23",
+"c. c #8072766B6739",
+"v. c #74E778D86480",
+"b. c #4C3186301C7A",
+"n. c #517082481F07",
+"m. c #4FB383C9233A",
+"M. c #55068B1F2455",
+"N. c #5A7482A32677",
+"B. c #5E2E8A712BE0",
+"V. c #5A3F85642AAA",
+"C. c #6A7F85712B12",
+"Z. c #645A8183336E",
+"A. c #65D5897C353E",
+"S. c #65658B2D39AF",
+"D. c #69118C1835E2",
+"F. c #6BF182763A39",
+"G. c #615491162F4A",
+"H. c #679292C2373E",
+"J. c #6A288A7345EC",
+"K. c #6D67933E40B6",
+"L. c #731F895D4558",
+"P. c #806486BF52EE",
+"I. c #784C9122480E",
+"U. c #71B9913955DE",
+"Y. c #759C810A6412",
+"T. c #7E1A8E9B70BB",
+"R. c #878F0F000EBE",
+"E. c #876D0FF41017",
+"W. c #809118EA17A1",
+"Q. c #83FD1D4019D3",
+"!. c #8C761382134E",
+"~. c #89A618471863",
+"^. c #85A9159A1491",
+"/. c #98540E7B0DD3",
+"(. c #9CCC156E146A",
+"). c #92A0176E167C",
+"_. c #85B92C321978",
+"`. c #88E827A91E00",
+"'. c #99E422211A83",
+"]. c #822E34FF2146",
+"[. c #8B7F3F0E32EA",
+"{. c #97C4254123B6",
+"}. c #A7F80E9E0E82",
+"|. c #A7E811AC1193",
+" X c #ACF914EF1474",
+".X c #A7E5193417C8",
+"XX c #B7A60F6F0FE5",
+"oX c #B5511808161C",
+"OX c #A47E212D1B91",
+"+X c #B66C21861F13",
+"@X c #A7983A612401",
+"#X c #822045CE3306",
+"$X c #8F1B5318331F",
+"%X c #906B7D0E3E5A",
+"&X c #8A8F6892370C",
+"*X c #82A162215413",
+"=X c #823B76CF5A2F",
+"-X c #95B07D7E54A4",
+";X c #88AC7C9A6D51",
+":X c #866A85944A5D",
+">X c #9D7983AF5630",
+",X c #84E3851E6FF0",
+"<X c #8E5E88EB76FD",
+"1X c #828999C6761A",
+"2X c #98488C816089",
+"3X c #98148C9C6C46",
+"4X c #982F8A8B7817",
+"5X c #844BA3E74427",
+"6X c #9E46A5766FF2",
+"7X c #A9E7946C7102",
+"8X c #B8FA8E586EA0",
+"9X c #AB7B85C366E3",
+"0X c #A2D8A76E4F68",
+"qX c #C2C3966F6CF9",
+"wX c #CB7D933173DD",
+"eX c #D4F8AA9C718D",
+"rX c #ED89BB757C15",
+"tX c #8EE18E5B8BDE",
+"yX c #99949310823F",
+"uX c #9EB89DE89B62",
+"iX c #ABE792898673",
+"pX c #A6389EEE93A3",
+"aX c #B6B899B5870E",
+"sX c #AAF4A87299BA",
+"dX c #B341B7F79B9F",
+"fX c #BAE9BAAFAEB3",
+"gX c #C70BA66F8B67",
+"hX c #C2C9A52191AB",
+"jX c #C820B3359390",
+"kX c #D9ACB46383F0",
+"lX c #D1EFBE239BC5",
+"zX c #D615C11B987B",
+"xX c #C6D3C19EBA7F",
+"cX c #DC2EC7A5AF34",
+"vX c #E68CC2408E4D",
+"bX c #EBF6C47A8CC7",
+"nX c #F03DD0E69FE9",
+"mX c #F537CE8194D5",
+"MX c #F65ED2EB9D3B",
+"NX c #EC5BC879902B",
+"BX c #EE71D028A38E",
+"VX c #E49CD198BD27",
+"CX c #F640D561A4F5",
+"ZX c #F715D7FFAAA9",
+"AX c #F819DA6FAF4A",
+"SX c #FA69DDF5B4C6",
+"DX c #F6AFDC1BB627",
+"FX c #D5B9D60FCF91",
+"GX c #DC27D935D5E5",
+"HX c #E172E0DBDF54",
+"JX c #F423E084CFC3",
+"KX c #F7E4E5ABCD32",
+"LX c #F38BEB63E0CA",
+"PX c #FA59EFF3E30E",
+"IX c #FD96FBB5F8C6",
+"UX c #E590E63EE4BA",
+"YX c None",
+/* pixels */
+"YXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYX",
+"YXYXYXYXYX            YXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYXYX",
+"YXYXYXYX    . + +                 YXYXYXYXYXYXYXYXYXYXYXYXYXYXYX",
+"YXYXYX    @ c ' ' 2         .                 YXYXYXYXYXYXYXYXYX",
+"YXYXYX  $ ! s.w.i.q         #       , .       YXYXYXYXYXYXYXYXYX",
+"YXYX    e W > > i.<       . , .   # 9 #       YXYXYXYXYXYXYXYXYX",
+"YX    & q . O u ' +       , h #   , 3         YXYXYXYXYXYXYXYXYX",
+"YX    7 : .   8 s         1 j #   1 ,         YXYXYXYXYXYXYXYXYX",
+"YX  + 7 + + . c v         0 E # . k 1         YXYXYXYXYXYXYXYXYX",
+"YX  $ 2 . & + 7 b 6 $ ; O c $.! ~ V.a             YXYXYXYXYXYXYX",
+"YX  @ 2 + = + . v ( Q 3.v.f.@.@.Z.S.d.2 = 6 @         YXYXYXYXYX",
+"YX  ; 7 + = &   i p.Q #.U.3.[ ` 8.%.=.+.! W c e : .       YXYXYX",
+". + + 6 2 = * X t.f.f...[ yXsXtX<Xf.s.t.: . : d +.~ 2         YX",
+"+ 2 * 2 s a * X z.3.O.o.c.GXpXfXtXT.a.;         ; s ~ :         ",
+"+ 6 6 6 6 7 % @ Y.Z.@.%.v.UXFXxXHXdXv               d d.q o     ",
+". + + + % +   + t.A.&.&.9.,X,X4XyXg.>               p r.f.k .   ",
+"      4 5 5 % ; f.D.:.4.Z.&.%.:XL.k .             : a.f f ' 7   ",
+"  % m J E.!.J S L.H.` ^ Z.V.m.B.D.x O             ~ v.@ O i.' > ",
+"  5 R././.|..X`.:.I.*XD s.6.X.M.G.=.u           + ..1Xy.t.f.s + ",
+"o N }.}.K K +X{.) 5.=XZ D >.#.m.M.B.7.i . .       c J.U.p.y +   ",
+"o B }.).H K OX`.].@X#XS *XiX3XS.M.M.5.o.( Q t .   + 2 2 +       ",
+"o N }.~.H L U U (.+X XQ.aXKXhX-XC.b.V.L.:.:.` $                 ",
+"o 5 !.|.XXXX(.(.~.(. XK #XwXVXgX%X5.n.S.9.0. .@                 ",
+"  o 5 N F (. X.X).^.'.Y $X8XLXcX>X5.| | G.D...@                 ",
+"      7 E A t I _..X`.P &XeXBXnXlX6X4.4.0.8.v                 YX",
+"< & < } &.j R ;.P _.`.G -.0XBXAXSXlX:Xq.:Xg.f         YXYXYXYXYX",
+"+ * w  .N.E ~ >X<.G P F 1.5XBXZXmXrXkXeXpXu.w.        YXYXYXYXYX",
+"    9 N.N.{ ' fXk.I T ,.h.3XkXZXmXbXvX4Xx.u @         YXYXYXYXYX",
+"    < $.$.@.E W y >.l.sXe.r.qXCXmXmXmX7X;             YXYXYXYXYX",
+"    + z X.$.$.{ j j x +.X 8 9XDXCXmXmXbXj.%         YXYXYXYXYXYX",
+"      # z  .} { E { @.l   8 aXPXSXMXMXMXvX3Xd         YXYXYXYXYX",
+"        + r w w w w j k > e.hXIXKXAXmXMXAXSXjXp       YXYXYXYXYX"
+};
